### PR TITLE
add Yu-Gi-Oh! Duel Monsters and Dragonball GT

### PR DESCRIPTION
**Pull request by hollanbm**

### CODE REVIEW

File: series-tmdb.en.yaml [hunk 1]
```diff
@@ -2888,6 +2901,23 @@ entries:
...
 
+  - title: "Yu-Gi-Oh! Duel Monsters"
+    guid: plex://show/5d9c081dba6eb9001fb9c35c
+    # imdb: https://www.imdb.com/title/tt0247902/
+    # tmdb: https://www.themoviedb.org/tv/902
+    # tvdb: https://www.thetvdb.com/dereferrer/series/76894
+    seasons:
+      - season: 1
+        anilist-id: 481
+      - season: 2
```
Comment (reconman) [inline series-tmdb.en.yaml:2912]: Yugioh on TMDB only has 1 season

File: series-tmdb.en.yaml [hunk 2]
```diff
@@ -611,6 +611,19 @@ entries:
...
 
+  - title: "Dragon Ball GT"
+    guid: plex://show/5d9c080a02391c001f57f8f9
+    # imdb: https://www.imdb.com/title/tt0139774/
+    # tmdb: https://www.themoviedb.org/tv/12697
+    # tvdb: https://www.thetvdb.com/dereferrer/series/79275
+    seasons:
+      - season: 1
+        anilist-id: 225
+      - season: 2
```
Comment (reconman) [inline series-tmdb.en.yaml:622]: GT only has 1 season on TMDB

**Review by reconman — CHANGES_REQUESTED** [review]

File: series-tmdb.en.yaml (hunk 1 above)
Comment (hollanbm) [reply series-tmdb.en.yaml]: I've removed the additional season entries

File: series-tmdb.en.yaml (hunk 2 above)
Comment (hollanbm) [reply series-tmdb.en.yaml]: I've removed the additional season entries

**Review by reconman — APPROVED** [review]

**Pull request merged**